api/api.py

```diff
@@ -139,6 +141,11 @@ async def get_gazettes(
         title="Post tags of fragments of highlight.",
         description="Post tags of fragments of highlight. This is a list of strings (usually HTML tags) that will appear after the text which matches the query",
     ),
+    sort_by: Optional[str] = Query(
+        "date_by_descending",
+        title="Allow the user to define the order of the search results.",
+        description="Allow the user to define the order of the search results. The API should allow 3 types: relevance, date_by_descending, date_by_ascending",
+    ),
```

Comment:
Please, use a enum to limit the possible value in the query. I'm able to insert any value in the field. You can take a look in the latest changes for the cities endpoints. It uses the enum to define the possible values for the cities levels.

Reply:
https://github.com/okfn-brasil/querido-diario-api/blob/main/api/api.py#L44-L57

Reply:
done :smile: